app/utils/colors.ts

```diff
@@ -0,0 +1,6 @@
+export const colors = {
+  COLORS: {
+    CONCRETE: '#f3f3f3',
+    CONIFER: '#AADD33',
+  },
+};
```

Comment:
You can use `COLORS` from `themes` dir.
Also, name your constants with uppercase, so `colors` -> `COLORS`